include/cotila/matrix/math.h

```diff
@@ -16,6 +18,11 @@ constexpr matrix<T, N, M> transpose(const matrix<T, N, M> &m) {
   return transposed;
 }
 
+template <typename T, std::size_t M, std::size_t N>
+constexpr matrix<T, N, M> hermitian(const matrix<T, N, M> &m) {
+  return transpose(elementwise([](const T t) { return cotila::conj(t); }, m));
```

Comment:
I'm thinking maybe it should be just `return transpose(conj(m))` where conj is a matrix elementwise conj function.

Reply:
Fair. I personally think it should work with the scalar math function but that was giving me compiler errors. I assume that's what you mean?